syntax/syntax.go

```diff
@@ -378,9 +378,8 @@ type TupleExpr struct {
 func (x *TupleExpr) Span() (start, end Position) {
 	if x.Lparen.IsValid() {
 		return x.Lparen, x.Rparen
-	} else {
```

Comment:
Please revert this edit too: neither value of the condition is more or less "normal" than the other.